plugins/module_utils/ansible_freeipa_module.py

```diff
@@ -497,6 +497,49 @@ def module_params_get_lowercase(module, name, allow_empty_list_item=False):
     return value
 
 
+def module_params_get_type(
```

Comment:
The `module_params_get_type` is misleading. Maybe something like `module_params_get_with_type` would be better.